site/docs/plugins/media-group.md

```diff
@@ -0,0 +1,84 @@
+# Media Groups (built-in)
```

Comment:
Why is it Media Groups here but the filename is media-group.md?

Reply:
It is also mentioned as “media group” in the first sentence.

Reply:
That's the terminology built-in plugins use. The session plugin deals with sessions. The keyboard plugin deals with keyboards. The media group plugin deals with media groups.

Reply:
Thanks. I just figured that out by reading the docs.